### ISSUE TITLE
--filename, scraped user name in output file?

### ISSUE THREAD

**Issue by blasphemite** [drawrowfly/tiktok-scraper#197]
I'm wondering if it's possible to use the `--filename,-f` option in the terminal to include a username when scraping video URLs from a list?

I haven't figured out how to do this with the --filename option, so I'm wondering if it's possible.

`tiktok-scraper from-file /path/to/link_list.txt 5 -s -d --filepath /scraped` on Ubuntu 18.04 is specifically the command I'm running, where link_list.txt is a list of URLs. This saves videos with the video ID (i think) as the filename.

**Comment by drawrowfly** [comment]
Not sure what exactly you mean

**Comment by AlphaBetaWhisky** [comment]
Lets say I download videos from "Username123", It would be nice to be able to set the username as the filename including the signature. 

From:
87623459876234598.mp4

To:
Username123_87623459876234598.mp4. 

And if possible make it work when you use from_file tag so that each file starts with the username that is associated to that user.

**Comment by drawrowfly** [comment]
I think this is unnecessary because when you download videos from a user profile then a new folder with the name of user is being created where all his videos are being saved

Also pull request are welcome

**Comment by blasphemite** [comment]
> I think this is unnecessary because when you download videos from a user profile then a new folder with the name of user is being created where all his videos are being saved

Right, but I mean when downloading videos from URLs, which doesn't include the username anywhere in the output.

**Comment by drawrowfly** [comment]
Got it. Will add it to a to do list
